content/03.text-mining.md

```diff
@@ -21,34 +21,67 @@
+Other approaches have used CNNs and variants of CNNs to extract relationship-asserting sentences [@doi:10.1177/0165551516673485; @doi:10.1093/database/bay073; @doi:10.1101/730085].
+Just like RNNs, these networks perform well when a millions amount of data is present [@arxiv:1707.02968]. 
+Future approaches that use CNNs or Recurrent neural networks should consider solutions to obtaining these large quantities of data through means such as weak supervision [@doi:10.3115/1690219.1690287], semi-supervised learning [@doi:10.2200/S00196ED1V01Y200906AIM006] or using pre-trained networks via transfer learning [@doi:10.1109/TKDE.2009.191; @doi:10.1186/s40537-016-0043-6].
+
+Semi-supervised learning [@doi:10.2200/S00196ED1V01Y200906AIM006] and weak supervision [@doi:10.3115/1690219.1690287] are techniques that can construct large datasets for machine learning classifiers to use. 
+Semi-supervised learning consist of combining labeled data with unlabeled data to extract relationships.
+For example, one study used a variational auto encoder with a LSTM network to extract protein-protein interactions from pubmed abstracts and full text [@arxiv:1901.06103v1].
+This is an elegant solution to handle the small dataset problem, but requires labeled data to start. 
+The dependency on labeled data makes finding under-studied relationships difficult as one would need to find or construct examples of the missing relationships in the beginning.
+
+Weak or distant supervision takes a different approach that uses noisy or even erroneous labels to train classifiers [@doi:10.3115/1690219.1690287; @doi:10.1093/bioinformatics/btv476; @doi:10.14778/3157794.3157797; @doi:10.1145/3209889.3209898].
+Under this paradigm sentences are labeled based on their mention pair being present (positive) or absent (negative) in a database.
+Once these labels are obtained a machine learning classifier can now be trained to predict sentences [@doi:10.3115/1690219.1690287].
+For example, Thomas et al. [@raw:Thomas2011LearningTE] used distant supervision to train a support vector machine to extract sentences mentioning protein-protein interactions (ppi). 
+Their SVM model achieved comparable performance against a baseline model; however, the noise generated via distant supervision was difficult to eradicate [@raw:Thomas2011LearningTE].
+Some approaches have attempted to fix this noise problem [@arxiv:1805.09927; @arxiv:1805.09929; @doi:10.18653/v1/W17-2323], but it still remains an open problem to be solved.
+Overall, semi-supervised learning and weak supervision provide promising results in terms of relation extraction and future approaches should consider using those paradigms to train machine learning classifiers.
+
+| Dataset | Type of Sentences |
+| --- | --- |
+| [@doi:10.1016/j.artmed.2004.07.016] | PPI |
```

Comment:
I think you're going to want to provide names for these - not just numerical references.